osi_sensorviewconfiguration.proto

```diff
@@ -729,6 +729,53 @@ message CameraSensorViewConfiguration
     //
     repeated WavelengthData wavelength_data = 11;
 
+    // Indicates if and how the the pixel data is ordered
+    //
+    // By default, the pixel data is is left-to-right, top-to-bottom. 
+    // 
+    // \note for rotations of the pixel data, use the camera coordinate system.
+    //
+    optional PixelOrder pixel_order = 12;
+
+    // Pixel layout 
+    //
+    // Pixel layout documents the order of pixels in the \c image_data
+    // in CameraSensorView. 
+    //
+    // \note this enum does not contain an entry to do mirroring upside down
+    // and left-to-right at the same time, because this is equivalent to a
+    // 180-degree rotation, which should be indicated in the sensor coordinate
+    // system.
+    //
+    enum PixelOrder
+    {
+        // Pixel order is unknown (must not be used).
+        //
+        PIXEL_ORDER_UNKNOWN = 0;
+
+        // Known pixel order that is not supported by OSI.
+        // Consider proposing an additional format if using
+        // \c #PIXEL_ORDER_OTHER.
+        //
+        PIXEL_ORDER_OTHER = 1;
+        
+        // Pixel data is not mirrored (Default).
+        // Pixels are ordered left-to-right, top-to-bottom
+        //
+        PIXEL_ORDER_DEFAULT = 2;
```

Comment:
Outcome Sensor Modeling Group meeting 25.01.2023: Place PIXEL_ORDER_DEFAULT on position 0 and remove PIXEL_ORDER_UNKNOWN

Reply:
The unknown is a mandatory field in OSI due to the fact that it is an enum.

Reply:
we would have to change the rules for enum definitions then?

Reply:
We (Pierre, Lukas and I) were aware of that guideline in OSI but discussed that it would not make sense to ever use unknown in this context as this is just a "technical" enum describing the pixel order that applies to the image data. It is not some kind of classification for which the guideline makes sense. I think there already were some other places in OSI where it was decided that the unknown/other guideline does not have to be enforced, especially when describing technical stuff.